Dockerfile.web

```diff
@@ -0,0 +1,36 @@
+ARG DOCKER_IMAGE=image-registry.openshift-image-registry.svc:5000/e1e498-tools/wps-web-base
+
+# PHASE 1 - build static html.
+# Pull from local registry - we can't pull from docker due to limits.
+# see https://catalog.redhat.com/software/containers/ubi8/nodejs-14/5ed7887dd70cc50e69c2fabb for details
+FROM registry.access.redhat.com/ubi8/nodejs-16 as static
+
+# Switch to root user for package installs
+USER 0
+
+ADD web .
+RUN npm i yarn && yarn install --frozen-lockfile
+# RUN yarn run build
+
+# Switch back to default user
+USER 1001
+
+# while testing stuff out:
+# docker build -f ./Dockerfile.web --tag=wps/web .
+# -p 8080:3000 - "map port 3000 in the container, to 8080 in the host"
+# docker run -p 3000:3000 wps/web
```

Comment:
Is this intended to be commented out?

Reply:
Yes - we don't have the actual build in place - so this is very temporary just helping me. This entire file is a placeholder.